webviz_config/_docs/static/README.md

```diff
@@ -0,0 +1,57 @@
+# Webviz introduction
+
+## Usage {docsify-ignore}
+
+Assuming you have a configuration file `your_config.yml`,
+there are two main usages of `webviz`:
+
+```bash
+webviz build your_config.yml
+```
+and
+```bash
+webviz build your_config.yml --portable ./some_output_folder
+python ./some_output_folder/webviz_app.py
+```
+
+The latter is useful when one or more plugins included in the configuration need to do
+some time-consuming data aggregation on their own before presenting it to the user
+(then the time-consuming part will be done in the `build` step, and you can run your
+created application as many time as you want afterwards, with as little waiting
+time as possible).
+
+The `--portable` way also has the benefit of creating a :whale: Docker setup for your
+application - ready to be deployed to e.g. a cloud provider.
+
```

Comment:
This is in risk of getting outdated since we have something in the official `README.md` as well on GitHub. Maybe we can split it out into a file `USAGE.md` an use it both places.